code/controllers/subsystem/shuttle.dm

```diff
@@ -242,7 +242,8 @@ SUBSYSTEM_DEF(shuttle)
 				return
 	if(world.time - SSticker.round_start_time >= 2 HOURS) //auto call the shuttle after 2 hours 
 		emergency_no_recall = TRUE //no recalling after 2 hours
-		if(emergency.timeLeft(1) > SSsecurity_level.current_security_level.shuttle_call_time_mod)
+		var/call_time = SSshuttle.emergency_call_time * SSsecurity_level.current_security_level.shuttle_call_time_mod * SSshuttle.emergency?.engine_coeff
+		if(emergency.timeLeft(1) > call_time)
```

Comment:
You should probably fix this for the crew death autocall as well

Reply:
no i dont wanna